Data/ByteString/Internal.hs

```diff
@@ -142,11 +174,18 @@ import GHC.Ptr                  (Ptr(..), castPtr)
 -- "Data.ByteString.Char8" it can be interpreted as containing 8-bit
 -- characters.
 --
-data ByteString = PS {-# UNPACK #-} !(ForeignPtr Word8) -- payload
-                     {-# UNPACK #-} !Int                -- offset
+data ByteString = BS {-# UNPACK #-} !(ForeignPtr Word8) -- payload
+                     -- {-# UNPACK #-} !Int                -- offset
```

Comment:
is there a reason for this line to remain?

Reply:
nope